### PR TITLE
Update Default Display Name

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Participant's display name returns `nil` if not set.

[#108473648]

### CODE REVIEW

File: app/models/participant.rb [hunk 1]
```diff
@@ -91,11 +91,7 @@ def self.inactive
   }
 
   def display_name
```
Comment (ericcf) [inline app/models/participant.rb]: Why does this method need to be overridden now?

Comment (michaelwehrley) [inline app/models/participant.rb]: Good point (or question I guess).  I forgot that AR has that an attribute as a default method.

**Comment by ericcf** [comment]
Lines of code deleted = win! :neckbeard:

**Comment by michaelwehrley** [comment]
heck yeah! :sagittarius:

**Pull request merged**